community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -505,3 +505,10 @@ variable "instance_properties" {
   type        = any
   default     = null
 }
+
+
+variable "enable_maintenance_reservation" {
+  type        = bool
+  description = "Enables slurm reservation for scheduled maintenance."
+  default     = true
```

Comment:
Shall it be `false` by default? At least until we publish documentation )

Reply:
as per best practise, we should keep `true` as default. But since this is behaviour change (all future deployments will have reservation for maintenance) keeping false atm and updating later is good way to go.